app/views/organizations/edit.html.erb

```diff
@@ -64,12 +64,12 @@
                   <%= f.input_field :zipcode, class: "form-control", placeholder: "zipcode" %>
                 <% end %>
 
-                <%= f.input :reminder_day, wrapper: :input_group do %>
+                <%= f.input :reminder_day, wrapper: :input_group, label: 'Reminder day (Day an e-mail reminder is sent to partners to submit requests)' do %>
                   <span class="input-group-text"><i class="fa fa-calendar"></i></span>
                   <%= f.input_field :reminder_day, class: "form-control", placeholder: "Reminder day" %>
                 <% end %>
 
-                <%= f.input :deadline_day, wrapper: :input_group do %>
+                <%= f.input :deadline_day, wrapper: :input_group, label: 'Deadline day (Final day to submit requests)' do %>
```

Comment:
Do you think it makes sense from a UX perspective to include information about it appearing in the next month under the new conditions?

Reply:
Honestly I think it's more intuitive now that it can work this way. I would add somewhere the fact that the day can't be after 28.

Reply:
In pursuit of better UX, I think I can cook something better up with JS.

Reply:
Right, now we have this:

![image](https://user-images.githubusercontent.com/22677/150675828-767466ab-5b84-413c-acc2-244a1b616cf6.png)
![image](https://user-images.githubusercontent.com/22677/150675832-e2c75b0c-fe6c-4e9a-be91-3f280a5d9f0f.png)
![image](https://user-images.githubusercontent.com/22677/150675850-dfdf0e32-91d1-4ddb-8eeb-066f55987cd8.png)